index.html

```diff
@@ -931,6 +931,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg</code></td>
   </tr>
+  <tr><th> Uptime Robot status: </th>
+    <td><img src='https://img.shields.io/uptimerobot/status/u956-afus321g565fghr519.svg?maxAge=2592000' alt=''/></td>
```

Comment:
Yes indeed on the API documentation they don't provide an monitor api key example.
Btw i asked a question about the API rate limit and they said :

> We don't apply a hard limit but monitor "bad usage" and warn if that happens. That almost never happened and no worries for that for such a legitimate usage.

So i will include an example with one of my monitor